Argo/Functions/flatReduce.swift

```diff
@@ -1,4 +1,4 @@
-public func flatReduce<S: SequenceType, U>(sequence: S, initial: U, combine: (U, S.Generator.Element) -> Decoded<U>) -> Decoded<U> {
+public func flatReduce<S: SequenceType, U>(sequence: S, initial: U, @noescape combine: (U, S.Generator.Element) -> Decoded<U>) -> Decoded<U> {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 142 characters